cmd/crank/xpkg/build.go

```diff
@@ -98,12 +98,12 @@ type buildCmd struct {
 	builder *xpkg.Builder
 	root    string
 
-	Output        string   `optional:"" short:"o" help:"Path for package output."`
-	Controller    string   `help:"Controller image used as base for package."`
-	ControllerTar string   `help:"Path to tar file, an alternative to Controller." xor:"controller" type:"existingfile"`
-	PackageRoot   string   `short:"f" help:"Path to package directory." default:"."`
-	ExamplesRoot  string   `short:"e" help:"Path to package examples directory." default:"./examples"`
-	Ignore        []string `help:"Paths, specified relative to --package-root, to exclude from the package."`
+	Output              string   `optional:"" short:"o" help:"Path for package output."`
+	RuntimeBaseImage    string   `help:"Name of the Docker image to be used as base for the package, e.g. 'xpkg.upbound.io/user/repo:v0.0.1'. Overwritten by '--runtime-base-image-tar'." `
```

Comment:
```suggestion
	RuntimeBaseImage    string   `help:"Name of the Docker image to be used as base for the package, typically runtime image, e.g. 'xpkg.upbound.io/user/repo:v0.0.1'. Overwritten by '--runtime-base-image-tar'." `
```

Reply:
FWIW, I am not feeling great about `--runtime-base-image`. I believe it should be either `--base-image` if we name it per implementation or `--runtime-image` if we name it per expected usage. `--runtime-base-image` looks like an odd mix of both views.

So, either:

```go
	BaseImage    string   `help:"Name of the Docker image to be used as the base layer for the package, typically runtime image'. Overwritten by '--base-image-tar'." `
```

or 

```go
	RuntimeImage    string   `help:"Name of the Runtime Docker image to be included into package image as the base layer, applicable for packages with a runtime (e.g. Providers and Functions)'. Overwritten by '--runtime-image-tar'." `
```

I have a slight preference over the former.

Reply:
I agree, I would vote for --base-image, as I feel users mostly care and is important for them to understand that a single image is going to be produced and used if they did not specify differently in the meta definition

Reply:
My vote is for `--runtime-image`. ;) It's more specific - we're building a Package around/containing a runtime, not just any old base image.

Reply:
I don't really get what you mean by "contains a runtime". Runtime used like that sounds to me as the Java or Python runtime, but I guess it's not what you mean. I would get saying it's "containing an image you can actually run", but your phrasing makes me doubt my interpretation.

However, we could postpone this discussion to when we get an agreement on https://github.com/crossplane/crossplane/issues/4783, as if we end up with `crossplane <noun> build`, we could decide to use specific flags meaningful for each package type.

Reply:
A runtime is what we're running with to describe "the code that powers a package". i.e.:

* A Provider controller is a Provider's runtime.
* A Function server/runner is a Function's runtime.

I know runtime is an overloaded term, but I think it's too late in the game to debate whether it makes sense. We're already committed via the "runtime config" concept, which configures how these things are deployed.

Reply:
Runtime config still made sense to me without this explanation, I read that as "the configuration to run a package" (for the kind of packages it make sense to do so).